src/menu/src/MenuItem.js

```diff
@@ -1,15 +1,20 @@
-import React, { memo, forwardRef, useCallback, useMemo } from 'react'
+import React, { memo, forwardRef, useCallback } from 'react'
+import cx from 'classnames'
 import PropTypes from 'prop-types'
 import Box from 'ui-box'
-import { useStyleConfig } from '../../hooks'
+import { useClickable, useStyleConfig } from '../../hooks'
 import { IconWrapper } from '../../icons/src/IconWrapper'
 import { Pane } from '../../layers'
-import safeInvoke from '../../lib/safe-invoke'
-import { pseudoSelectors } from '../../table/src/TableRow'
+import { pseudoSelectors as rowPseudos } from '../../table/src/TableRow'
 import { Text } from '../../typography'
 
 const noop = () => {}
 
+const pseudoSelectors = {
+  ...rowPseudos,
```

Comment:
Sigh - would love to decouple the two here, but I guess out of scope for the PR

Reply:
Yeah not a fan that it was imported to begin with. We can duplicate it.